Section renumbering and more consistent use of .BR.  Add language
describing use of SMF.  Fix a few small errors.  Some of this is obviously
Solaris-specific, but the rest could be submitted upstream.

--- swift-2.7.0/doc/manpages/account-server.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/account-server.conf.5	2016-08-08 19:02:27.970894135 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH account-server.conf 5 "8/26/2011" "Linux" "OpenStack Swift"
+.TH account-server.conf 4 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B account-server.conf
-\- configuration file for the openstack-swift account server
+\- configuration file for the OpenStack Swift account server
 
 
 
@@ -346,11 +346,11 @@ requested by delay_reaping.
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-account-server and
-also Openstack-Swift as a whole can be found at
+also OpenStack Swift as a whole can be found at
 .BI http://swift.openstack.org/admin_guide.html
 and
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-account-server(1),
+.BR swift-account-server (1)
--- swift-2.7.0/doc/manpages/container-server.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/container-server.conf.5	2016-08-08 19:02:27.971606335 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH container-server.conf 5 "8/26/2011" "Linux" "OpenStack Swift"
+.TH container-server.conf 4 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B container-server.conf
-\- configuration file for the openstack-swift container server
+\- configuration file for the OpenStack Swift container server
 
 
 
@@ -379,11 +379,11 @@ Internal client config file path.
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-container-server and
-also Openstack-Swift as a whole can be found at
+also OpenStack Swift as a whole can be found at
 .BI http://swift.openstack.org/admin_guide.html
 and
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-container-server(1)
+.BR swift-container-server (1)
--- swift-2.7.0/doc/manpages/dispersion.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/dispersion.conf.5	2016-08-08 19:02:27.972256225 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH dispersion.conf 5 "8/26/2011" "Linux" "OpenStack Swift"
+.TH dispersion.conf 4 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B dispersion.conf
-\- configuration file for the openstack-swift dispersion tools 
+\- configuration file for the OpenStack Swift dispersion tools 
 
 .SH SYNOPSIS
 .LP
@@ -55,7 +55,7 @@ The default is 'publicURL'.
 .IP "\fBkeystone_api_insecure\fR"
 The default is false.
 .IP "\fBswift_dir\fR"
-Location of openstack-swift configuration and ring files
+Location of OpenStack Swift configuration and ring files
 .IP "\fBdispersion_coverage\fR"
 Percentage of partition coverage to use. The default is 1.0.
 .IP "\fBretries\fR"
@@ -100,13 +100,13 @@ Whether to run the object report. The de
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-dispersion utilities and
-also Openstack-Swift as a whole can be found at 
+also OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/admin_guide.html#cluster-health
 and 
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-dispersion-report(1),
-.BR swift-dispersion-populate(1)
+.BR swift-dispersion-report (1),
+.BR swift-dispersion-populate (1)
 
--- swift-2.7.0/doc/manpages/object-expirer.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/object-expirer.conf.5	2016-08-08 19:02:27.972835975 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH object-expirer.conf 5 "03/15/2012" "Linux" "OpenStack Swift"
+.TH object-expirer.conf 4 "03/15/2012" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B object-expirer.conf
-\- configuration file for the openstack-swift object exprier daemon  
+\- configuration file for the OpenStack Swift object exprier daemon  
 
 
 
@@ -206,12 +206,12 @@ Path to recon cache directory. The defau
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-object-expirer and
-also Openstack-Swift as a whole can be found at 
+also OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/admin_guide.html 
 and 
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-proxy-server.conf(5),
+.BR proxy-server.conf (4)
 
--- swift-2.7.0/doc/manpages/object-server.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/object-server.conf.5	2016-08-08 19:02:27.973565170 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH object-server.conf 5 "8/26/2011" "Linux" "OpenStack Swift"
+.TH object-server.conf 4 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B object-server.conf
-\- configuration file for the openstack-swift object server
+\- configuration file for the OpenStack Swift object server
 
 
 
@@ -510,11 +510,11 @@ will try to use object-replicator's rsyn
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-object-server and
-also Openstack-Swift as a whole can be found at
+also OpenStack Swift as a whole can be found at
 .BI http://swift.openstack.org/admin_guide.html
 and
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-object-server(1),
+.BR swift-object-server (1)
--- swift-2.7.0/doc/manpages/proxy-server.conf.5.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/proxy-server.conf.5	2016-08-08 19:02:27.974300095 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH proxy-server.conf 5 "8/26/2011" "Linux" "OpenStack Swift"
+.TH proxy-server.conf 4 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B proxy-server.conf
-\- configuration file for the openstack-swift proxy server
+\- configuration file for the OpenStack Swift proxy server
 
 
 
@@ -1036,10 +1036,10 @@ The default is 'x-container-read, x-cont
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-proxy-server and
-also Openstack-Swift as a whole can be found at
+also OpenStack Swift as a whole can be found at
 .BI http://swift.openstack.org/admin_guide.html
 and
 .BI http://swift.openstack.org
 
 .SH "SEE ALSO"
-.BR swift-proxy-server(1)
+.BR swift-proxy-server (1)
--- swift-2.7.0/doc/manpages/swift-account-auditor.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-account-auditor.1	2016-08-08 19:02:27.974810365 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-account-auditor 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-account-auditor 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-account-auditor 
-\- Openstack-swift account auditor
+\- OpenStack Swift account auditor
 
 .SH SYNOPSIS
 .LP
-.B swift-account-auditor 
+.B /usr/lib/swift/swift-account-auditor
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -50,13 +50,25 @@ The options are as follows:
 .RE
 .PD
 .RE
+
+.LP
+The account auditor should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+account auditor is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-account-auditor
+.fi
+.in -2
     
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-account-auditor 
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 .SH "SEE ALSO"
-.BR account-server.conf(5)
+.BR account-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-account-info.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-account-info.1	2016-08-08 19:02:27.975290530 -0700
@@ -14,12 +14,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-account-info 1 "3/22/2014" "Linux" "OpenStack Swift"
+.TH swift-account-info 1 "3/22/2014" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-account-info
-\- Openstack-swift account-info tool
+\- Openstack Swift account-info tool
 
 .SH SYNOPSIS
 .LP
@@ -50,11 +50,11 @@ It will then return several information
     
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at 
+More documentation about Openstack Swift can be found at 
 .BI http://swift.openstack.org/index.html
 
 .SH "SEE ALSO"
 
-.BR swift-container-info(1),
-.BR swift-get-nodes(1),
-.BR swift-object-info(1)
+.BR swift-container-info (1),
+.BR swift-get-nodes (1),
+.BR swift-object-info (1)
--- swift-2.7.0/doc/manpages/swift-account-reaper.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-account-reaper.1	2016-08-08 19:02:27.975805060 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-account-reaper 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-account-reaper 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-account-reaper
-\- Openstack-swift account reaper
+\- OpenStack Swift account reaper
 
 .SH SYNOPSIS
 .LP
-.B swift-account-reaper 
+.B /usr/lib/swift/swift-account-reaper
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -56,14 +56,25 @@ The options are as follows:
 .PD
 .RE
 
-    
+.LP
+The account reaper should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the account reaper
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-account-reaper
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
-.BI swift-object-auditor 
-and also about Openstack-Swift as a whole can be found at 
+.BI swift-account-reaper 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR account-server.conf(5)
+.BR account-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-account-replicator.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-account-replicator.1	2016-08-08 19:02:27.976300255 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-account-replicator 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-account-replicator 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-account-replicator 
-\- Openstack-swift account replicator
+\- OpenStack Swift account replicator
 
 .SH SYNOPSIS
 .LP
-.B swift-account-replicator 
+.B /usr/lib/swift/swift-account-replicator
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -58,14 +58,25 @@ The options are as follows:
 .PD 
 .RE
     
-   
+.LP
+The account replicator should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+account replicator is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-account-replicator
+.fi
+.in -2
+    
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-account-replicator
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR account-server.conf(5)
+.BR account-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-account-server.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-account-server.1	2016-08-08 19:02:27.976796260 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-account-server 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-account-server 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-account-server
-\- Openstack-swift account server
+\- OpenStack Swift account server
 
 .SH SYNOPSIS
 .LP
-.B swift-account-server
+.B /usr/lib/swift/swift-account-server
 [CONFIG] [-h|--help] [-v|--verbose]
 
 .SH DESCRIPTION 
@@ -33,15 +33,26 @@ The Account Server's primary job is to h
 are stored as sqlite database files, and replicated across the cluster similar to how
 objects are. 
 
+.LP
+The account server should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the account server
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-account-server
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-account-server
-and also about Openstack-Swift as a whole can be found at 
+and also about Openstack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 and 
 .BI http://docs.openstack.org
 
-
 .SH "SEE ALSO"
-.BR account-server.conf(5)
+.BR account-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-container-auditor.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-auditor.1	2016-08-08 19:02:27.977281630 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-auditor 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-container-auditor 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-auditor 
-\- Openstack-swift container auditor
+\- OpenStack Swift container auditor
 
 .SH SYNOPSIS
 .LP
-.B swift-container-auditor 
+.B /usr/lib/swift/swift-container-auditor
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -50,15 +50,26 @@ The options are as follows:
 .RE
 .PD
 .RE
-     	
-    
+
+.LP
+The container auditor should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+container auditor is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-container-auditor
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-container-auditor 
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR container-server.conf(5)
+.BR container-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-container-info.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-info.1	2016-08-08 19:02:27.977764165 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-info 1 "3/20/2013" "Linux" "OpenStack Swift"
+.TH swift-container-info 1 "3/20/2013" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-info
-\- Openstack-swift container-info tool
+\- Openstack Swift container-info tool
 
 .SH SYNOPSIS
 .LP
@@ -57,9 +57,9 @@ It will then return several information
     
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at 
+More documentation about Openstack Swift can be found at 
 .BI http://swift.openstack.org/index.html
 
 .SH "SEE ALSO"
-.BR swift-get-nodes(1),
-.BR swift-object-info(1)
+.BR swift-get-nodes (1),
+.BR swift-object-info (1)
--- swift-2.7.0/doc/manpages/swift-container-replicator.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-replicator.1	2016-08-08 19:02:27.978254170 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-replicator 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-container-replicator 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-replicator 
-\- Openstack-swift container replicator
+\- OpenStack Swift container replicator
 
 .SH SYNOPSIS
 .LP
-.B swift-container-replicator 
+.B /usr/lib/swift/swift-container-replicator
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -57,15 +57,26 @@ The options are as follows:
 .RE
 .PD
 .RE
+
+.LP
+The container replicator should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+container replicator is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-container-replicator
+.fi
+.in -2
     
-   
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-container-replicator
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR container-server.conf(5)
+.BR container-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-container-server.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-server.1	2016-08-08 19:02:27.978779845 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-server 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-container-server 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-server
-\- Openstack-swift container server
+\- OpenStack Swift container server
 
 .SH SYNOPSIS
 .LP
-.B swift-container-server
+.B /usr/lib/swift/swift-container-server
 [CONFIG] [-h|--help] [-v|--verbose]
 
 .SH DESCRIPTION 
@@ -35,11 +35,22 @@ are stored as sqlite database files, and
 objects are. Statistics are also tracked that include the total number of objects, and 
 total storage usage for that container.
 
+.LP
+The container server should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+container server is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-container-server
+.fi
+.in -2
+    
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-container-server
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 and 
 .BI http://docs.openstack.org
@@ -47,4 +58,5 @@ and
 .LP 
 
 .SH "SEE ALSO"
-.BR container-server.conf(5)
+.BR container-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-container-sync.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-sync.1	2016-08-08 19:02:27.979295920 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-sync 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-container-sync 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-sync
-\- Openstack-swift container sync
+\- OpenStack Swift container sync
 
 .SH SYNOPSIS
 .LP
@@ -44,7 +44,7 @@ newer rows since the last sync will trig
 .LP
 More in depth documentation in regards to 
 .BI swift-container-sync
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/overview_container_sync.html
 and 
 .BI http://docs.openstack.org
@@ -52,4 +52,4 @@ and
 .LP 
 
 .SH "SEE ALSO"
-.BR container-server.conf(5)
+.BR container-server.conf (4)
--- swift-2.7.0/doc/manpages/swift-container-updater.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-container-updater.1	2016-08-08 19:02:27.979802740 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-container-updater 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-container-updater 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-container-updater
-\- Openstack-swift container updater
+\- OpenStack Swift container updater
 
 .SH SYNOPSIS
 .LP
-.B swift-container-updater 
+.B /usr/lib/swift/swift-container-updater
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -59,14 +59,26 @@ The options are as follows:
 .RE
 .PD
 .RE
-       
+
+.LP
+The container updater should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the
+container updater is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-container-updater
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-container-updater
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR container-server.conf(5)
+.BR container-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-dispersion-populate.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-dispersion-populate.1	2016-08-08 19:02:27.980351620 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-dispersion-populate 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-dispersion-populate 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-dispersion-populate
-\- Openstack-swift dispersion populate 
+\- OpenStack Swift dispersion populate 
 
 .SH SYNOPSIS
 .LP
@@ -110,12 +110,12 @@ $ swift-dispersion-populate
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-dispersion utilities and
-also Openstack-Swift as a whole can be found at 
+also OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/admin_guide.html#cluster-health
 and 
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-dispersion-report(1),
-.BR dispersion.conf (5)
+.BR swift-dispersion-report (1),
+.BR dispersion.conf (4)
--- swift-2.7.0/doc/manpages/swift-dispersion-report.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-dispersion-report.1	2016-08-08 19:02:27.980871430 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-dispersion-report 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-dispersion-report 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-dispersion-report
-\- Openstack-swift dispersion report 
+\- OpenStack Swift dispersion report 
 
 .SH SYNOPSIS
 .LP
@@ -134,12 +134,12 @@ $ swift-dispersion-report
 .SH DOCUMENTATION
 .LP
 More in depth documentation about the swift-dispersion utilities and
-also Openstack-Swift as a whole can be found at 
+also OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/admin_guide.html#cluster-health
 and 
 .BI http://swift.openstack.org
 
 
 .SH "SEE ALSO"
-.BR swift-dispersion-populate(1),
-.BR dispersion.conf (5)
+.BR swift-dispersion-populate (1),
+.BR dispersion.conf (4)
--- swift-2.7.0/doc/manpages/swift-get-nodes.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-get-nodes.1	2016-08-08 19:02:27.981361915 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-get-nodes 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-get-nodes 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-get-nodes
-\- Openstack-swift get-nodes tool
+\- OpenStack Swift get-nodes tool
 
 .SH SYNOPSIS
 .LP
@@ -72,14 +72,14 @@ where the container resides by using the
 
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at 
+More documentation about Openstack Swift can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 
 .SH "SEE ALSO"
 
-.BR swift-account-info(1),
-.BR swift-container-info(1),
-.BR swift-object-info(1),
-.BR swift-ring-builder(1)
+.BR swift-account-info (1),
+.BR swift-container-info (1),
+.BR swift-object-info (1),
+.BR swift-ring-builder (1)
--- swift-2.7.0/doc/manpages/swift-init.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-init.1	2016-08-08 19:02:27.981914380 -0700
@@ -15,22 +15,54 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-init 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-init 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-init
-\- Openstack-swift swift-init tool
+\- Openstack Swift swift-init tool
 
 .SH SYNOPSIS
 .LP
-.B swift-init
- <server> [<server> ...] <command> [options]
+.B /usr/lib/swift/swift-init
+<server> [<server> ...] <command> [options]
  
 .SH DESCRIPTION 
 .PP
+The suite of servers implementing OpenStack Swift should be managed using
+the Solaris Service Management Facility, \fBsmf\fR(7).  There are sixteen
+services:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-account-auditor
+svc:/application/openstack/swift/swift-account-reaper
+svc:/application/openstack/swift/swift-account-replicator
+svc:/application/openstack/swift/swift-account-server
+svc:/application/openstack/swift/swift-container-auditor
+svc:/application/openstack/swift/swift-container-replicator
+svc:/application/openstack/swift/swift-container-server
+svc:/application/openstack/swift/swift-container-sync
+svc:/application/openstack/swift/swift-container-updater
+svc:/application/openstack/swift/swift-object-auditor
+svc:/application/openstack/swift/swift-object-expirer
+svc:/application/openstack/swift/swift-object-replicator
+svc:/application/openstack/swift/swift-object-server
+svc:/application/openstack/swift/swift-object-updater
+svc:/application/openstack/swift/swift-proxy-server
+svc:/application/openstack/swift/swift-replicator-rsync
+.fi
+.in -2
+.LP
+Each service is described more completely in its corresponding manual page.
+Note that there are no dependencies between the services, although most of
+them should be started in groups, even if the entire suite isn't necessary.
+.LP
+Although the \fBswift-init\fR tool can be used to manage the services,
+doing so is not supported on Solaris.
+.LP
 The swift-init tool can be used to initialize all swift daemons available as part of
-openstack-swift. Instead of calling individual init scripts for each 
+OpenStack Swift. Instead of calling individual init scripts for each 
 swift daemon, one can just use swift-init. With swift-init you can initialize 
 just one swift service, such as the "proxy", or a combination of them. The tool also 
 allows one to use the keywords such as "all", "main" and "rest" for the <server> argument.
@@ -119,8 +151,26 @@ allows one to use the keywords such as "
 
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at 
+More documentation about OpenStack Swift can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 
+.SH SEE ALSO
+.BR swift-account-auditor (1),
+.BR swift-account-reaper (1),
+.BR swift-account-replicator (1),
+.BR swift-account-server (1),
+.BR swift-container-auditor (1),
+.BR swift-container-replicator (1),
+.BR swift-container-server (1),
+.BR swift-container-sync (1),
+.BR swift-container-updater
+.BR swift-object-auditor (1),
+.BR swift-object-expirer (1),
+.BR swift-object-replicator (1),
+.BR swift-object-server (1),
+.BR swift-object-updater (1),
+.BR swift-proxy-server (1),
+.BR swift-replicator-rsync (1),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-object-auditor.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-auditor.1	2016-08-08 19:02:27.982404670 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-auditor 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-object-auditor 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-auditor 
-\- Openstack-swift object auditor
+\- OpenStack Swift object auditor
 
 .SH SYNOPSIS
 .LP
-.B swift-object-auditor 
+.B /usr/lib/swift/swift-object-auditor
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once] [-z|--zero_byte_fps]
 
 .SH DESCRIPTION 
@@ -56,15 +56,26 @@ The options are as follows:
 .RE
 .PD
 .RE
-    
-    
+
+.LP
+The object auditor should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the object auditor
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-object-auditor
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-object-auditor 
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR object-server.conf(5)
+.BR object-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-object-expirer.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-expirer.1	2016-08-08 19:02:27.982889395 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-expirer 1 "3/15/2012" "Linux" "OpenStack Swift"
+.TH swift-object-expirer 1 "3/15/2012" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-expirer
-\- Openstack-swift object expirer
+\- OpenStack Swift object expirer
 
 .SH SYNOPSIS
 .LP
-.B swift-object-expirer 
+.B /usr/lib/swift/swift-object-expirer
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -57,18 +57,28 @@ The options are as follows:
 .RE
 .PD
 .RE
-    
-   
+
+.LP
+The object expirer should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the object expirer
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-object-expirer
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-object-expirer
 can be foud at 
 .BI http://swift.openstack.org/overview_expiring_objects.html
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR object-expirer.conf(5)
-
+.BR object-expirer.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-object-info.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-info.1	2016-08-08 19:02:27.983363875 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-info 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-object-info 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-info
-\- Openstack-swift object-info tool
+\- OpenStack Swift object-info tool
 
 .SH SYNOPSIS
 .LP
@@ -48,11 +48,11 @@ It will then return several information
     
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at 
+More documentation about Openstack Swift can be found at 
 .BI http://swift.openstack.org/index.html
 
 .SH "SEE ALSO"
 
-.BR swift-account-info(1),
-.BR swift-container-info(1),
-.BR swift-get-nodes(1)
+.BR swift-account-info (1),
+.BR swift-container-info (1),
+.BR swift-get-nodes (1)
--- swift-2.7.0/doc/manpages/swift-object-replicator.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-replicator.1	2016-08-08 19:02:27.983847760 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-replicator 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-object-replicator 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-replicator 
-\- Openstack-swift object replicator
+\- OpenStack Swift object replicator
 
 .SH SYNOPSIS
 .LP
-.B swift-object-replicator 
+.B /usr/lib/swift/swift-object-replicator 
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -57,15 +57,26 @@ The options are as follows:
 .RE
 .PD
 .RE
-    
-   
+
+.LP
+The object replicator should be enabled and disabled using the Solaris
+Service Management Facility, \fBsmf\fR(7).  The service FMRI for the object
+replicator is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-object-replicator
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-object-replicator
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR object-server.conf(5)
+.BR object-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-object-server.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-server.1	2016-08-08 19:02:27.984336625 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-server 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-object-server 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-server
-\- Openstack-swift object server.
+\- OpenStack Swift object server.
 
 .SH SYNOPSIS
 .LP
-.B swift-object-server
+.B /usr/lib/swift/swift-object-server
 [CONFIG] [-h|--help] [-v|--verbose]
 
 .SH DESCRIPTION 
@@ -40,15 +40,27 @@ served. A deletion is also treated as a
 ".ts", which stands for tombstone). This ensures that deleted files are replicated 
 correctly and older versions don't magically reappear due to failure scenarios.
 
+.LP
+The object server should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the object server
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-object-server
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-object-server
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 and 
 .BI http://docs.openstack.org
 
 
 .SH "SEE ALSO"
-.BR object-server.conf(5)
+.BR object-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-object-updater.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-object-updater.1	2016-08-08 19:02:27.984816205 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-object-updater 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-object-updater 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-object-updater
-\- Openstack-swift object updater
+\- OpenStack Swift object updater
 
 .SH SYNOPSIS
 .LP
-.B swift-object-updater
+.B /usr/lib/swift/swift-object-updater
 [CONFIG] [-h|--help] [-v|--verbose] [-o|--once]
 
 .SH DESCRIPTION 
@@ -64,15 +64,26 @@ The options are as follows:
 .RE
 .PD 
 .RE
-    
-    
+
+.LP
+The object updater should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the object updater
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-object-updater
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-object-updater
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR object-server.conf(5)
+.BR object-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-orphans.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-orphans.1	2016-08-08 19:02:27.985268230 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH swift-orphans 1 "3/15/2012" "Linux" "OpenStack Swift"
+.TH swift-orphans 1 "3/15/2012" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B swift-orphans
-\- Openstack-swift orphans tool
+\- OpenStack Swift orphans tool
 
 .SH SYNOPSIS
 .LP
@@ -65,6 +65,6 @@ The options are as follows:
 
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at
+More documentation about OpenStack Swift can be found at
 .BI http://swift.openstack.org/index.html
 
--- swift-2.7.0/doc/manpages/swift-proxy-server.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-proxy-server.1	2016-08-08 19:02:27.985740415 -0700
@@ -15,16 +15,16 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-proxy-server 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-proxy-server 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-proxy-server 
-\- Openstack-swift proxy server.
+\- OpenStack Swift proxy server.
 
 .SH SYNOPSIS
 .LP
-.B swift-proxy-server
+.B /usr/lib/swift/swift-proxy-server
 [CONFIG] [-h|--help] [-v|--verbose]
 
 .SH DESCRIPTION 
@@ -38,13 +38,25 @@ and route there instead. When objects ar
 streamed directly through the proxy server to or from the user the proxy server does 
 not spool them.
 
+.LP
+The proxy server should be enabled and disabled using the Solaris Service
+Management Facility, \fBsmf\fR(7).  The service FMRI for the proxy server
+is:
+.sp
+.in +2
+.nf
+svc:/application/openstack/swift/swift-proxy-server
+.fi
+.in -2
+
 .SH DOCUMENTATION
 .LP
 More in depth documentation in regards to 
 .BI swift-proxy-server
-and also about Openstack-Swift as a whole can be found at 
+and also about OpenStack Swift as a whole can be found at 
 .BI http://swift.openstack.org/index.html
 
 
 .SH "SEE ALSO"
-.BR proxy-server.conf(5)
+.BR proxy-server.conf (4),
+.BR smf (7)
--- swift-2.7.0/doc/manpages/swift-recon.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-recon.1	2016-08-08 19:02:27.986285440 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"
-.TH swift-recon 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-recon 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME
 .LP
 .B swift-recon
-\- Openstack-swift recon middleware cli tool
+\- OpenStack Swift recon middleware cli tool
 
 .SH SYNOPSIS
 .LP
@@ -124,7 +124,7 @@ cronjob to run the swift-recon-cron scri
 
 .SH DOCUMENTATION
 .LP
-More documentation about Openstack-Swift can be found at
+More documentation about OpenStack Swift can be found at
 .BI http://swift.openstack.org/index.html
 Also more specific documentation about swift-recon can be found at
 .BI http://swift.openstack.org/admin_guide.html#cluster-telemetry-and-monitoring
@@ -132,6 +132,6 @@ Also more specific documentation about s
 
 
 .SH "SEE ALSO"
-.BR object-server.conf(5),
+.BR object-server.conf (4)
 
 
--- swift-2.7.0/doc/manpages/swift-ring-builder.1.~1~	2016-03-25 02:42:55.000000000 -0700
+++ swift-2.7.0/doc/manpages/swift-ring-builder.1	2016-08-08 19:02:27.986766640 -0700
@@ -15,12 +15,12 @@
 .\" See the License for the specific language governing permissions and
 .\" limitations under the License.
 .\"  
-.TH swift-ring-builder 1 "8/26/2011" "Linux" "OpenStack Swift"
+.TH swift-ring-builder 1 "8/26/2011" "OpenStack" "OpenStack Swift"
 
 .SH NAME 
 .LP
 .B swift-ring-builder
-\- Openstack-swift ring builder
+\- OpenStack Swift ring builder
 
 .SH SYNOPSIS
 .LP
@@ -210,7 +210,7 @@ calls when no rebalance is needed but yo
 
 .SH DOCUMENTATION
 .LP
-More in depth documentation about the swift ring and also Openstack-Swift as a 
+More in depth documentation about the swift ring and also OpenStack Swift as a 
 whole can be found at 
 .BI http://swift.openstack.org/overview_ring.html, 
 .BI http://swift.openstack.org/admin_guide.html#managing-the-rings 
